<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BINAIRO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="mainGrid">
      <item row="8" column="7">
       <widget class="QPushButton" name="checkPushButton">
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="9">
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>0</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="boardSizeSpinBox">
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="randomRadioButton">
        <property name="text">
         <string>Generate random</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QLabel" name="errorLabel">
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLineEdit" name="inputLine">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Input seed</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="inputRadioButton">
        <property name="text">
         <string>Fill from input</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" rowspan="2" colspan="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="infoButton">
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
      <item row="8" column="5" colspan="2">
       <widget class="QLabel" name="winLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="manualRadioButton">
        <property name="text">
         <string>Set manually</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="setBoardButton">
        <property name="text">
         <string>Set board</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Board size:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="playPauseButton">
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="styleSheet">
         <string notr="true">font: 300 18pt &quot;Segoe UI Light&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>B I N A I R O</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" rowspan="7" colspan="4">
       <widget class="QFrame" name="boardFrame">
        <property name="minimumSize">
         <size>
          <width>512</width>
          <height>512</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(40, 29, 3, 255), stop:0.0852273 rgba(46, 28, 7, 255), stop:0.159091 rgba(73, 47, 10, 255), stop:0.238636 rgba(48, 36, 17, 255), stop:0.318182 rgba(57, 31, 9, 255), stop:0.386364 rgba(39, 25, 6, 255), stop:0.477273 rgba(62, 36, 15, 255), stop:0.568182 rgba(48, 33, 13, 255), stop:0.642045 rgba(53, 32, 13, 255), stop:0.695 rgba(41, 25, 14, 255), stop:0.721591 rgba(73, 49, 34, 255), stop:0.789773 rgba(43, 25, 4, 255), stop:0.857955 rgba(58, 37, 16, 255), stop:0.982955 rgba(41, 26, 8, 255), stop:1 rgba(41, 28, 3, 255));</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="cellGrid">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
